app/src/main/res/layout/nav_list.xml

```diff
@@ -1,74 +1,119 @@
-        android:layout_centerVertical="true"
-        android:background="?android:attr/listDivider"
-        tools:background="@android:color/holo_red_dark" />
+            <TextView
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_vertical"
+                android:layout_weight="1"
+                android:gravity="center_vertical"
+                android:text="@string/feed_is_filtered"
+                android:textColor="?android:attr/textColorSecondary"
+                android:textSize="@dimen/text_size_small"
+                tools:background="@android:color/holo_green_light" />
+        </LinearLayout>
+
+        <View
+            android:id="@+id/divider"
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:background="?android:attr/listDivider"
+            tools:background="@android:color/holo_red_dark" />
+
+        <LinearLayout
+            android:id="@+id/nav_settings"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/listitem_iconwithtext_height"
+            android:background="?attr/selectableItemBackground"
+            android:contentDescription="@string/settings_label"
+            android:focusable="true"
+            android:orientation="horizontal">
+
+            <ImageView
+                android:id="@+id/imgvCover"
+                android:layout_width="@dimen/thumbnail_length_navlist"
+                android:layout_height="@dimen/thumbnail_length_navlist"
+                android:layout_marginStart="@dimen/listitem_icon_leftpadding"
+                android:layout_marginLeft="@dimen/listitem_icon_leftpadding"
+                android:layout_marginTop="4dp"
+                android:layout_marginBottom="4dp"
+                android:adjustViewBounds="true"
+                android:cropToPadding="true"
+                android:importantForAccessibility="no"
+                android:padding="8dp"
+                android:scaleType="centerCrop"
+                android:src="?attr/ic_settings"
+                tools:background="@android:color/holo_orange_dark"
+                tools:src="@android:drawable/sym_def_app_icon" />
+
+            <TextView
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_vertical"
+                android:layout_marginStart="16dp"
+                android:layout_marginLeft="16dp"
+                android:layout_weight="1"
+                android:gravity="center_vertical"
+                android:text="@string/settings_label"
+                android:textColor="?android:attr/textColorPrimary"
+                android:textSize="@dimen/text_size_navdrawer"
+                tools:background="@android:color/holo_green_light" />
+        </LinearLayout>
+    </LinearLayout>
```

Comment:
I think it would be cool if the message could be displayed above the subscriptions list (directly below the separator line). That way, you do not run into the situation that subscriptions are cut off by the filter message (see screenshot)

<img width="500" src="https://user-images.githubusercontent.com/5811634/88030048-0e33b480-cb3b-11ea-9d67-673239ebe9e0.png" />

The starting point for this change would be `NavListAdapter.getSectionDividerView`. The dialog can then be shown in `NavDrawerFragment.onItemClick`.